tests/causal-consistency/causal-consistency-011.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+Causal consistency: $clusterTime is not sent in commands to unsupported deployments
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('STANDALONE'); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+require_once __DIR__ . "/../utils/observer.php";
+
+(new CommandObserver)->observe(
+    function() {
+        $manager = new MongoDB\Driver\Manager(STANDALONE);
```

Comment:
Note: libmongoc does gossip the cluster time to a standalone if the user explicitly advances the cluster time on the session object (see: [CDRIVER-2439](https://jira.mongodb.org/browse/CDRIVER-2439)); however, that's outside the scope of the causal consistency spec test so we're not testing that here.